src/SprykerEco/Zed/AmazonQuicksight/Communication/Plugin/User/QuicksightUserUserExpanderPlugin.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\AmazonQuicksight\Communication\Plugin\User;
+
+use Generated\Shared\Transfer\UserCollectionTransfer;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+use Spryker\Zed\UserExtension\Dependency\Plugin\UserExpanderPluginInterface;
+
+/**
+ * @method \SprykerEco\Zed\AmazonQuicksight\AmazonQuicksightConfig getConfig()
+ * @method \SprykerEco\Zed\AmazonQuicksight\Business\AmazonQuicksightFacadeInterface getFacade()
+ * @method \SprykerEco\Zed\AmazonQuicksight\Communication\AmazonQuicksightCommunicationFactory getFactory()
+ */
+class QuicksightUserUserExpanderPlugin extends AbstractPlugin implements UserExpanderPluginInterface
```

Comment:
Maybe we can make an exception from our conventions and name this plugin `QuicksightUserExpanderPlugin` to avoid duplication in the class name?